Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -165,6 +178,32 @@ void DMainWindow::OnOpen()
 		StartGame(filename);
 }
 
+void DMainWindow::OnBrowse()
+{
+	std::string path = ShowFolderDialog().toStdString();
+	std::vector<std::string>* iso_folder = &SConfig::GetInstance().m_ISOFolder;
+	if (!path.empty()){
```

Comment:
`iso_folder` is declared as `std::vector<std::string>*` right up there, so making the iterator `auto` should be fine.